### DISCUSSION TITLE
adding custom menu icons to the Shell component

### DISCUSSION THREAD

**Discussion by ToniJ123** [sqlpage/SQLPage#670] [Ideas]
Hi,

This popped out from the mysterious world of internet.

https://github.com/pchemguy/SQLPage-Demo/tree/main/Locale%20Theme%20Menu

Could these features be implemented to current shell component as native and official features? Color icons, usage of local icons and not denying still usage of tabler.io icons etc.

What you think about?

**Comment by lovasoa** [comment]
Hi @ToniJ123 ! This is a great demo built by @pchemguy . You may be interested in the previous discussions he opened on this forum: https://github.com/sqlpage/SQLPage/discussions?discussions_q=author%3Apchemguy , including this one where he presented the demo you are referring to: https://github.com/sqlpage/SQLPage/discussions/465

The official shell component has evolved quite a bit since this demo, so this work cannot be directly integrated today. However, if someone wants to open a pull request cleanly implementing it or to sponsor this work, I'll be glad to accept it !
